### ISSUE TITLE
Støtte wcag 1.4.4 (øke tekststørrelse)

### ISSUE THREAD

**Issue by idaons** [navikt/nav-dekoratoren_legacy#770]
Kravet sier at "Tekst kan bli endret til 200% størrelse uten tap av innhold eller funksjon."
Men ser at hvis jeg øker font-størrelse i browser til 200 % får blant annet ikke menyen plass, og noe tekst går utenfor lenkepaneler.

Når WCAG 2.1 innføres, blir det også et krav om at man skal kunne zoome 400% 

Relatert eksempel:  https://nav-it.slack.com/archives/C7MANSGLS/p1603363933163800 

Vi bør gå igjennom css'en og endre fra px til em/rem, og se om dette fikser problemet.

**Comment by idaons** [comment]
Påbegynt her: https://github.com/navikt/nav-dekoratoren/tree/px-to-relative-units

Mangler testing og justering på elementer som blir off

**Comment by Laier68** [comment]
Jeg har bistått med litt testbistand i denne saken.
Mission fra Ida O:  
Det skal oppføre seg likt som i prod (ca, trenger ikke være heelt på milimeter likt). Gjerne teste litt forskjellige browsere også, og bare sjekke at ingenting ser veldig rart ut. 

1) Har foretatt en generell sjekk av dekoratør (header/footer) for å sjekke alt ser ut til å fungere.
    Noen somå issues mht plassering av elementer ellers OK. 
 2) Browsere/OS er/ Plattformer
   a) Jeg har tatt en kjapp fra laptop win10 mht siste versjon av browserene Firefox, Vivaldi, Opera, Edge (87) og selvf. også 
       Chrome. Der oppleves dekoratøren OK.  
       Men opplevde krøll i Edge 44--> Det er fikset OK.
   b)Sjekket  Mac Big sur / Safari 14 og Mac Catilina / Safari 13.  Dekoratøren oppleves OK i begge de.
   c) Sjekket IOS 14 med Safari, IOS 13 med Safari og IOS 12 med Safari. Det oppleves OK.
   d) Sjekket Android 10 med chrome  og Android 9 med chrome. Det oppleves OK.

**Comment by idaons** [comment]
Det ligger en pr klar på dette, men jeg er usikker på hvor mye invirkning den vil ha for team som bruker dekoratøren, så jeg har ikke merget den. Det vil være litt små forskjeller i høyde og bredde på dekoratøren avhengig av hva slags fontstørrelse appen den vises i bruker.

**Comment by theasteen** [comment]
Noen apper setter size på rota... kan ikke skille font size satt i nettleser eller css-en. Få endrer font-size i nettleser. Tall??
I teorien alvorlige feil... 

@chrisarnesen Sjekker med UU-teamet hva vi bør gjøre.

**Comment by ghost** [comment]
Det er ikke noe krav å tilby at man kan endre fontstørrelse i innstillinger på telefon, så lenge Zoom er på plass.
Tatt en Zoom test, det er mulig å Zoome inn til 200 og 400%, innhold skalerer som det skal.
